src/videojs-contrib-hls.js

```diff
@@ -278,8 +279,42 @@ export default class HlsHandler extends Component {
       let seekable;
 
       if (!updatedPlaylist) {
+        let selectedPlaylist = this.selectPlaylist();
+
+        if (!selectedPlaylist.attributes.AUDIO) {
+          // from safari defaults
+          this.tech_.audioTracks().addTrack(new AudioTrack({
+            enabled: true,
+            id: '1',
+            kind: 'main',
+            tech: this.tech_
```

Comment:
Another question, do audioTracks and videoTracks actually need a reference to the tech inside them? TextTracks do because that's how they update activeCues but TextTracks are also not generally created by hand but via helper methods.
Maybe only TextTracks need to require techs?